src/components/tooltip/tooltip.spec.js

```diff
@@ -430,8 +430,8 @@ describe('when off is true', () => {
 
     // should not be visible
     expect(queryByText('What kind of bear is best?')).not.toBeInTheDocument();
-    // should not remove title
-    expect(button).toHaveProperty('title', 'What kind of bear is best?');
+    // should not have title
+    expect(button).not.toHaveProperty('title', 'What kind of bear is best?');
```

Comment:
```suggestion
    expect(button).not.toHaveAttribute('title');
```

I think it's a bit more safe like this, as we just want to check that the `title` attribute is not set and not some specific title.